src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Windows/Markup/ArrayExtension.cs

```diff
@@ -48,27 +48,23 @@ public ArrayExtension(Array elements)
             Type = elements.GetType().GetElementType();
         }
 
-        ///<summary>
-        /// Called to Add an object as a new array item.  This will append the object to the end
-        /// of the array.
-        ///</summary>
-        ///<param name="value">
-        /// Object to add to the end of the array.
-        ///</param>
+        /// <summary>
+        /// Called to Add an object as a new array item. This will append the
+        /// object to the end of the array.
+        /// </summary>
+        /// <param name="value">Object to add to the end of the array.</param>
         public void AddChild(object value) => _arrayList.Add(value);
 
-        ///<summary>
-        /// Called to Add a text as a new array item.  This will append the object to the end
-        /// of the array.
-        ///</summary>
-        ///<param name="text">
-        /// Text to Add to the end of the array
-        ///</param> 
+        /// <summary>
+        /// Called to Add a text as a new array item. This will append the
+        /// object to the end of the array.
+        /// </summary>
+        /// <param name="text">Text to Add to the end of the array.</param> 
         public void AddText(string text) => AddChild(text);
 
         ///<summary>
         /// Get and set the type of array to be created when calling ProvideValue
-        ///</summary>
+        /// </summary>
```

Comment:
redundant space

Reply:
Visual Studio generates default document comment **with a space** between `///` and identifiers.

So it's not redundant, it should be more.